changelog/744.misc.rst

```diff
@@ -0,0 +1 @@
+Update annotation and description of ``options`` parameter of :func:`ui.ActionRow.add_select <disnake.ui.ActionRow.add_select>` to match :class:`ui.Select <disnake.ui.Select>`.
```

Comment:
This is a docs change, not a miscellaneous change, is it not?

Reply:
It's kind of both a typing change and a docs change, but adding two changelog fragments for something minor like this felt a bit excessive.